dir.props

```diff
@@ -58,7 +58,8 @@
     <ToolPackageExtractDir>$(ToolsDir)source-built/</ToolPackageExtractDir>
     <ToolsLocalDir>$(ProjectDir)tools-local/</ToolsLocalDir>
     <TaskDirectory>$(ToolsLocalDir)tasks/</TaskDirectory>
-    <DarcDll>$(ToolsLocalDir)arcade-services/artifacts/bin/Microsoft.DotNet.Darc/Release/netcoreapp3.0/Microsoft.DotNet.Darc.dll</DarcDll>
+    <DarcVersion>1.1.0-beta.19515.3</DarcVersion>
```

Comment:
Do you think we could do this in a `DarcVersion.txt` file like we used to do for the SDK and ILAsm so we wouldn't have to duplicate it?

Reply:
Yeah. That will surely be a neater way. I can do that 👍